backend/src/src-moderation/src/main/java/com/siemens/sw360/moderation/db/ModerationDatabaseHandler.java

```diff
@@ -260,6 +272,12 @@ public void createRequest(User user) {
         addOrUpdate(request);
     }
 
+    private String getDepartmentFromUserEmail(String userEmail) throws TException {
```

Comment:
I have a feeling that in this method you're doing the job of the `UserService` for it. Admittedly, it's only a two-liner job, so feel free to disagree.
Also, I'd call this method `getDepartmentByUserEmail`